pocs/observatory.py

```diff
@@ -485,6 +486,9 @@ def autofocus_cameras(self, camera_list=None, coarse=False):
 
         autofocus_events = dict()
 
+        if coarse is None:
+            coarse = not self._coarse_focus_done
```

Comment:
Seems like either this variable name is wrong or the default logic is wrong. Doesn't this end up doing a coarse focus every time *but* the first time? Also see setter below.

Reply:
I think it's doing something fairly sensible. `self._coarse_focus_done` is set to `False` in the constructor, so unless `coarse` has been set in the call to `autofocus_cameras` then `coarse` will get set to `True` here, and so it will do a coarse focus the first time `autofocus_cameras` is called.

Not clear to me that we actually want it to do this, though. Coarse focus is only needed in a pretty limited set of circumstances already, and will only be needed less with time.

Reply:
Sadly I think I wrote this code so I would hopefully understand it. Actually, I missed the `not` in the logic. Thanks.